src/py-opentimelineio/opentimelineio/adapters/cmx_3600.py

```diff
@@ -985,9 +985,8 @@ def __init__(
                     line.source_in.rate
                 )
             elif timing_effect.effect_name == "LinearTimeWarp":
-                value = clip.trimmed_range().duration.value / timing_effect.time_scalar
-                line.source_out = (
-                    line.source_in + opentime.RationalTime(value, rate))
+                value = clip.trimmed_range().duration.value * timing_effect.time_scalar - 1
```

Comment:
```suggestion
                value = (
                    clip.trimmed_range().duration.value * timing_effect.time_scalar 
                    - 1
                )
```

Reply:
I think that will fix the lint error.  Is the `-1` because this is an inclusive range?